src/NServiceBus.Metrics/TaskExtensions.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NServiceBus.Metrics
+{
+    using System.Threading.Tasks;
+
+    class TaskExtensions
+    {
+        public static Task Completed => Task.FromResult(0);
```

Comment:
Make the class static and assign the static field once. See https://github.com/ahofman/NServiceBus/blob/develop/src/NServiceBus.Core/TaskEx.cs#L19